qemu/tests/blockdev_inc_backup_nospace_with_bitmap_mode.py

```diff
@@ -0,0 +1,92 @@
+from provider.block_dirty_bitmap import get_bitmap_by_name
+from provider.blockdev_live_backup_base import BlockdevLiveBackupBaseTest
+from provider.job_utils import BLOCK_JOB_COMPLETED_EVENT
+from provider.job_utils import get_event_by_condition
+
+
+class BlockdevIncbkIncNospaceWithSyncMode(BlockdevLiveBackupBaseTest):
+    """
+    Do incremental backup to an image without enough space,
+    set bitmap mode to: on-success or always, check bitmap count
+    """
+
+    def __init__(self, test, params, env):
+        super(BlockdevIncbkIncNospaceWithSyncMode, self).__init__(test,
+                                                                  params,
+                                                                  env)
+        self._inc_bitmap_mode = params["inc_bitmap_mode"]
+        self._inc_bk_nodes = ["drive_%s" % t for t in self._target_images]
+
+    def _get_bitmap_count(self):
+        bm = get_bitmap_by_name(self.main_vm, self._source_nodes[0],
+                                self._bitmaps[0])
+        if bm:
+            if bm.get("count", 0) <= 0:
+                self.test.fail("Count of bitmap should be greater than 0")
+            return bm["count"]
+        else:
+            self.test.fail("Failed to get bitmap")
+
+    def do_incremental_backup(self):
+        self.count_before_incbk = self._get_bitmap_count()
```

Comment:
@zhencliu, on ppc host, the on-success test fails for the bitmap count obtained here is incorrect, because it's too quick to get the bitmap count after generating the inc file; if wait 10 seconds after generating the inc file, it can pass.

Reply:
hi @ngu-niny , I added a sleep when getting bitmap count.